test/generator/underscore_test.rb

```diff
@@ -19,5 +19,9 @@ def test_question_mark
         'unreadable_but_correctly_sized_inputs_return_?'
       )
     end
+
+    def test_fixnum_underscore
+      assert_equal '1_000_000', 1_000_000.underscore
```

Comment:
The tests should test from a "raw" number, such as `1000000` rather than one with underscores, otherwise why would the call be needed?  (Yes, it is needed to also convert it to a string... but this is not clear in the name, especially since the number already has underscores that it is being called on.)